apps/client/src/components/Conclusion/NewCheckerModal.tsx

```diff
@@ -47,15 +47,19 @@ const NewCheckerModal: React.FC<{
 
     if (checkerSlug) {
       const address = saveAddress ? sessionContext[slug].address : null;
-
+      const activeComponents = saveAddress
+        ? sections.QUESTIONS
+        : sections.LOCATION_INPUT;
+      const finishedComponents = saveAddress && sections.LOCATION_RESULT;
```

Comment:
Should this not be an array of sections.LOCATION_INPUT and sections.LOCATION_RESULT?

Reply:
This should just be location Result.